File: front/src/app/classes/transaction.ts
import { StatusTransaction } from '../enums/status-transaction.enum';
import { TypeTransaction } from '../enums/type-transaction.enum';

export class Transaction {
  public id: number;

  public status: StatusTransaction;

  public type: TypeTransaction;

  public clientName: string;

  public amount: number;
}

File: backend/Entities/Models/Transaction.cs
﻿using System.ComponentModel.DataAnnotations.Schema;
using Entities.Enums;

namespace Entities.Models
{
  [Table("Transaction")]
  public class Transaction
  {
    public int Id { get; set; }

    public EnumStatusTransaction Status { get; set; }

    public EnumTypeTransaction Type { get; set; }

    public string ClientName { get; set; }

    public double Amount { get; set; }
  }
}